react/src/pages/MainMenu/MainMenu.tsx

```diff
@@ -9,8 +11,35 @@ import designsafeLogo from '@hazmapper/assets/designsafe.svg';
 import nheriLogo from '@hazmapper/assets/nheri.png';
 
 const MainMenu = () => {
+  const location = useLocation();
+  const [message, contextHolder] = notification.useNotification();
+  const [onDeleteSuccess, setOnDeleteSuccess] = useState(
+    location.state?.onSuccess
+  );
+
+  useEffect(() => {
+    if (location.state?.onSuccess) {
+      setOnDeleteSuccess(true);
+    }
+  }, [location.state]);
+
+  useEffect(() => {
+    if (onDeleteSuccess) {
+      message.destroy();
+      message.success({
+        message: 'Success',
+        description: 'Your map was successfully deleted.',
+        placement: 'topRight',
```

Comment:
Thoughts on doing bottom-left notifications instead of top-right? We do bottom-left in designsafe

Reply:
I like that. That way the notifications aren't clashing with the modal if one opens while a success message is up.